### ISSUE TITLE
Character Animation Toolkit

### ISSUE THREAD

**Issue by Yusuf2k** [airbnb/lottie-web#2559]
@bodymovin 
I have used Character Animation Toolkit for the character animation
After Rendering the body doesnt seem to animate only head is animated i have also checked Hidden layers and Guide layers but it didnt work

here is the file
[Skeleton Walking.zip](https://github.com/airbnb/lottie-web/files/6638669/Skeleton.Walking.zip)

**Comment by bodymovin** [comment]
indeed, unfortunately character animation is not supported by lottie
